File: testers/grpc/java-spring/generated-and-checked-in/com/example/grpc/EchoService.java
package com.example.grpc;

/** Clean service interface for EchoService gRPC service */
public interface EchoService {
  ScalarTypes echoScalarTypes(ScalarTypes request);

  Customer echoCustomer(Customer request);

  Order echoOrder(Order request);

  Inventory echoInventory(Inventory request);

  Outer echoOuter(Outer request);

  OptionalFields echoOptionalFields(OptionalFields request);

  WellKnownTypesMessage echoWellKnownTypes(WellKnownTypesMessage request);

  PaymentMethod echoPaymentMethod(PaymentMethod request);

  Notification echoNotification(Notification request);
}

File: testers/grpc/kotlin-quarkus/generated-and-checked-in/com/example/grpc/EchoService.kt
package com.example.grpc

import io.smallrye.mutiny.Uni

/** Clean service interface for EchoService gRPC service */
interface EchoService {
  abstract fun echoCustomer(request: Customer): Uni<Customer>

  abstract fun echoInventory(request: Inventory): Uni<Inventory>

  abstract fun echoNotification(request: Notification): Uni<Notification>

  abstract fun echoOptionalFields(request: OptionalFields): Uni<OptionalFields>

  abstract fun echoOrder(request: Order): Uni<Order>

  abstract fun echoOuter(request: Outer): Uni<Outer>

  abstract fun echoPaymentMethod(request: PaymentMethod): Uni<PaymentMethod>

  abstract fun echoScalarTypes(request: ScalarTypes): Uni<ScalarTypes>

  abstract fun echoWellKnownTypes(request: WellKnownTypesMessage): Uni<WellKnownTypesMessage>
}